lib/block-supports/layout.php

```diff
@@ -58,12 +58,12 @@ function gutenberg_get_layout_style( $selector, $layout ) {
 			$style .= '}';
 
 			$style .= "$selector > .alignwide { max-width: " . esc_html( $wide_max_width_value ) . ';}';
-
 			$style .= "$selector .alignfull { max-width: none; }";
 		}
 
 		$style .= "$selector .alignleft { float: left; margin-right: 2em; }";
 		$style .= "$selector .alignright { float: right; margin-left: 2em; }";
+		$style .= "$selector > * + * { margin-top: var( --wp--style--block-gap ); margin-bottom: 0; }";
 	} elseif ( 'flex' === $layout_type ) {
 		$style  = "$selector {";
```

Comment:
In my tests the block gap works inside a group block with the default layout but not with the flex layout I think it is expected for now given the PR title. But I guess we can easily also add a rule on the flex layout that uses the --wp--style--block-gap variable.

Reply:
I actually added it to the client but forgot to add it to the server :)